src/devices/Bmp180/Bmp180.cs

```diff
@@ -0,0 +1,303 @@
+            int B5 = (X1 + X2);
+
+            // Pressure Calculations
+            int B6 = B5 - 4000;
+            X1 = ((short)_calibrationData.B2 * (B6 * B6 / (1 << 12))) / (1 << 11);
+            X2 = (short)_calibrationData.AC2 * B6 / (1 << 11);
+            int X3 = X1 + X2;
+            int B3 = ((((short)_calibrationData.AC1 * 4 + X3) << (short)Sampling.Standard) + 2) / 4;
+            X1 = (short)_calibrationData.AC3 * B6 / (1 << 13);
+            X2 = ((short)_calibrationData.B1 * (B6 * B6 / (1 << 12))) / (1 << 16);
+            X3 = ((X1 + X2) + 2) / (1 << 2);
+            int B4 = _calibrationData.AC4 * (X3 + 32768) / (1 << 15);
+            int B7 = (UP - B3) * (50000 >> (short)Sampling.Standard);
+
+            int p = 0;
+            if((uint)B7 < 0x80000000)
+            {                
+                p = (int)(((long)B7 * 2) / B4);                
+            }
+            else
+            {               
+                p = (int)(((long)B7 / B4) * 2);             
+            }
+            
+            X1 = (p / (1 << 8)) * (p / (1 << 8));
+            X1 = (X1 * 3038) / (1 << 16);
+            X2 = (-7357 * p) / (1 << 16);
+            p = p + (X1 + X2 + 3791) / (1 << 4);
+
+            return p;
+        }
+
+        /// <summary>
+        ///  Calculates the altitude in meters from the specified sea-level pressure(in hPa).
+        /// </summary>
+        /// <param name="seaLevelPressure" > 
+        ///  Sea-level pressure in hPa
+        /// </param>
+        /// <returns>
+        ///  Height in meters from the sensor
+        /// </returns>
+        public double ReadAltitude(double sealevel_pa = 101325.0)
+        {
+            double pressure = (double)ReadPressure();
+            double altitude = 44330.0 * (1.0 - Math.Pow((pressure / sealevel_pa), (1.0 / 5.255)));
+
+
+            return altitude;
+        }
+
+        /// <summary>
+        ///  Calculates the pressure at sealevel when given a known altitude in meter
+        /// </summary>
+        /// <param name="altitude" > 
+        ///  altitude in meters
+        /// </param>
+        /// <returns>
+        ///  Pressure in Pascals
+        /// </returns>
+        public double ReadSeaLevelPressure(double altitude = 0.0)
+        {
+            double pressure = (double)ReadPressure();
+            double p0 = pressure / Math.Pow((1.0 - (altitude / 44333.0)), 5.255);
+
+            return p0;
+        }
+
+        /// <summary>
+        ///  Reads raw temperatue from the sensor
+        /// </summary>
+        /// <returns>
+        ///  Raw temperature
+        /// </returns>
+        short ReadRawTemperature()
+        {
+            // Reads the raw (uncompensated) temperature from the sensor
+            _i2cDevice.Write(new[] { (byte)Register.CONTROL, (byte)Register.READTEMPCMD });            
+            // Wait 5ms
+            Thread.Sleep(5);

```

Comment:
Why?

Reply:
![timewait](https://user-images.githubusercontent.com/14133711/52926277-bcaa4900-32fa-11e9-8f33-658d9e9b8360.PNG)

Reply:
Can you add a comment referencing the data sheet/section number so we know where 5 ms came from?